src/utils.js

```diff
@@ -19,7 +19,7 @@ export const validateProductForm = (values, rules, isProductCodeValid) => {
   const errors = {};
 
   REQUIRED_FIELDS.forEach((field) => {
-    if (!values[field]) {
+    if (!values[field] & values[field]!= 0) {
```

Comment:
For type safety, it is recommended to use the strict inequality operator "!==", rather than the standard inequality operator "!=".

Reply:
Moreover, use logical AND (double ampersand &&).
`if (!values[field] && values[field] !== 0) {...}`